drizzlepac/haputils/svm_quality_analysis.py

```diff
@@ -516,6 +516,304 @@ def compare_ra_dec_crossmatches(hap_obj, json_timestamp=None, json_time_since_ep
+                                               "comparison catalog filename": "ECSV segment catalog filename",
+                                               "reference catalog length": "Number of entries in point catalog",
+                                               "comparison catalog length": "Number of entries in segment catalog",
+                                               "number of cross-matches": "Number of cross-matches between point and segment catalogs"},
+                                           units={"reference catalog filename": "unitless",
+                                                  "comparison catalog filename": "unitless",
+                                                  "reference catalog length": "unitless",
+                                                  "comparison catalog length": "unitless",
+                                                  "number of cross-matches": "unitless"})
+
+                    # Generate tables containing just "xcentroid_ref" and "ycentroid_ref" columns with only
+                    # the cross-matched reference sources
+                    matched_ref_coords = filtobj_dict[xmatch_ref_imgname]["sources"].copy()
+                    matched_ref_coords.keep_columns(['xcentroid_ref', 'ycentroid_ref'])
+
+                    matched_ref_coords = matched_ref_coords[matching_lines_ref]
+
+                    # store reference matched sources catalog
+                    diag_obj.add_data_item(matched_ref_coords, "Interfilter cross-matched reference catalog",
+                                           descriptions={"xcentroid_ref": "xcentroid_ref",
+                                                         "ycentroid_ref": "ycentroid_ref"},
+                                           units={"xcentroid_ref": "pixels", "ycentroid_ref": "pixels"})
+
+                    # write out ds9 region files if log level is 'debug'
+                    if log_level == logutil.logging.DEBUG:
+                        reg_filename = "{}_{}_ref_matches.reg".format(filtername_comp, filtername_ref)
+                        matched_ref_coords.write(reg_filename, format='ascii.csv')
+
+                    # Generate tables containing just "xcentroid_ref" and "ycentroid_ref" columns with only
+                    # the cross-matched comparision sources
+                    matched_comp_coords = filtobj_dict[imgname]["sources"].copy()
+                    matched_comp_coords.keep_columns(['xcentroid_ref', 'ycentroid_ref'])
+                    matched_comp_coords = matched_comp_coords[matching_lines_comp]
+
+                    # store comparison matched sources catalog
+                    diag_obj.add_data_item(matched_comp_coords,
+                                           "Interfilter cross-matched comparison catalog",
+                                           descriptions={"xcentroid_ref": "xcentroid_ref",
```

Comment:
Should these column names include '_ref' since they refer to the comparison image positions?  We really do not want these columns confused (or possibly over-written by or over-writing) the same columns from the reference data.